core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -136,6 +136,26 @@ private[spark] class Executor(
   // for fetching remote cached RDD blocks, so need to make sure it uses the right classloader too.
   env.serializerManager.setDefaultClassLoader(replClassLoader)
 
+  private val plugins: Seq[ExecutorPlugin] = {
+    val pluginNames = conf.get(EXECUTOR_PLUGINS)
+    if (pluginNames.nonEmpty) {
+      logDebug(s"Initializing the following plugins: ${pluginNames.mkString(", ")}")
+
+      // Before loading the executor plugins, we need to temporarily change the class loader
```

Comment:
This comment is actually not currently accurate unless you're running YARN. In this context, it should mention the "executor's user classpath", not how it's formed.